docs/index.html

```diff
@@ -105,104 +104,67 @@ <h1 class="au-header__heading">Example Site</h1>
 			  </div>
 		  </div>
 	  </nav>
-  
+
 
 	<!-- BANNER -->
 	<div class="banner au-body au-body--alt">
 		<div class="container-fluid">
 			<div class="row">
-				<div class="col-sm-6">
-					<h3>Understand user needs. Research to develop a deep knowledge of the users and their context for using the service.</h3>
-					<br/>
-					<button class="au-btn au-btn--secondary">
-						Examples
-					</button>
-					<button class="au-btn">
-						Get Started
-					</button>
+				<div class="col-sm-12 col-md-6">
+					<h2>Welcome to the design system starter site</h2>
+					<p>If you inspect the html for this page you will see that the <code>&lt;body&gt;</code> tag is using a class called <code>au-grid</code>. This class let's you use the design system grid which is based on 12 columns.</p>
+					<p>Use button classes such as <code>au-btn</code> to style your interactive calls to action.</p>
+					<a class="au-btn au-btn--secondary" href="https://designsystem.gov.au/components/grid-12/">Read about grid</a>
```

Comment:
I would reword to something like this:

If you inspect the HTML for this page, you will the the CSS class `au-grid` is applied to the `<body>` element. This CSS class lets you use the design system grid, which allows up to 12 columns across a page

thoughts?